### PR TITLE
Issue/refactor ecs task secrets

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

- Refactor `ecs task` to take in multiple aws secrets
- upgrade uk development
- upgrade india development
This helps things like

Screenshot 2024-09-05 at 12 32 39

This will helps #603 

## How Has This Been Tested?

- CI tests
- No actual changes in AWS things
- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: terraform/modules/services/ecs_task/variables.tf [hunk 1]
```diff
@@ -63,16 +58,27 @@ variable container-env_vars {
...
-}
-
 variable container-command {
   type = list(string)
   description = "Command to run in the container"
 }
 
+variable container-secrets {
+  type = list(object({
+    id = string
```
Comment (peterdudfield) [inline terraform/modules/services/ecs_task/variables.tf:68]: the id isnt strictly needed, but I quite like having it

**Comment by devsjc** [comment]
Looks good, my only comment is that I wonder if it ought still to be called `container-secret_vars`, as that is the fundamental utility of the variable - it's passing vars from a secret into a container. `container-secrets` suggests some other utility to me. Also, I see you've noted that `id` is redundant - what's the argument for including it?

**Comment by peterdudfield** [comment]
> Looks good, my only comment is that I wonder if it ought still to be called `container-secret_vars`, as that is the fundamental utility of the variable - it's passing vars from a secret into a container. `container-secrets` suggests some other utility to me. Also, I see you've noted that `id` is redundant - what's the argument for including it?

Ok
1. Yea good idea, to rename
2. probably only make it clear to the read what its for, but perhaps its extra lines of code and doesn't help that much. A comment would perhaps be better

**Comment by devsjc** [comment]
I guess I would argue that its clear from the `arn`, but I don't want to shout down something you like if it makes it easier to parse for you! As long as its clear in the description that `id` doesn't do anything of value I don't mind you keeping it!

**Review by devsjc — APPROVED** [review]

**Pull request merged**